src/map/zone.cpp

```diff
@@ -1032,9 +1032,8 @@ void CZone::CharZoneIn(CCharEntity* PChar)
         PChar->StatusEffectContainer->DelStatusEffectSilent(EFFECT_MOUNTED);
     }
 
-    if (PChar->m_Costume != 0)
+    if (PChar->StatusEffectContainer->HasStatusEffect(EFFECT_COSTUME))
     {
-        PChar->m_Costume = 0;
         PChar->StatusEffectContainer->DelStatusEffect(EFFECT_COSTUME);
```

Comment:
Should this be silent like the mounted effect removal?

Reply:
Probably, however, it's never a bad thing to check.
You can check it yourself though. I saw you opened a PR for the quadav costume.
Use the item, zone and check if there is a message in the chatlog, pls.

Reply:
> Probably, however, it's never a bad thing to check. You can check it yourself though. I saw you opened a PR for the quadav costume. Use the item, zone and check if there is a message in the chatlog, pls.

Looks like it is silently falling off:

![image](https://github.com/LandSandBoat/server/assets/166072302/17a1d909-5886-42de-822b-e3d1baa91862)

Updated to be silent.